lib/odbc/configure.in

```diff
@@ -46,7 +46,6 @@ AC_ARG_WITH(odbc,
 if test "$with_odbc" = "no"; then
 
    rm -f "$ERL_TOP/lib/odbc/SKIP"
-   echo "odbc disabled by user." > "$ERL_TOP/lib/odbc/SKIP"
```

Comment:
ODBCs configure was changed so that you would be able to reconfigure the odbc application without re-running the whole OTP configure, and that is probably how it ended up with duplicate info. Although it is practical to be able to do this ODBC is not truly standalone as it still has makefile dependencies on OTP.  I guess if you re-run the ODBC configure after the inital build it is not because you want to disable ODBC and hence it would be ok to remove this line.

Reply:
😄 I didn't understand if you ended up agreeing with the change, or not - but I guess you did.

This PR started by wanting to fix `kerl`'s output, where I was getting duplicate info. for stuff like ODBC (I'd disable it and then I'd get lines of output stating I'd disabled it - when I was already aware I did 😄). The same goes for `wx` and SSL-related stuff, the most "visible" ones in the logs I checkout out.

(as I stated before, though, this is a small step in the right direction, but probably not the walk).

Reply:
I ended up agreeing that it was ok to remove the "odbc disabled by user" from the ODBC configure, as if you are running configure only in the ODBC-application the reason will not be that you want to disable it.

Reply:
👍